### PR TITLE
Fixed broken xrefs

### PR DESCRIPTION
**Pull request by Hopeopolis**
Corrected broken xrefs found while reviewing mouseovers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,7 +272,8 @@ also of likely interest to founders and hiring managers:
 - [**Evaluating Equity Compensation**](#evaluating-equity-compensation): Whether you have equity now
   or will in the future, it is helpful to learn *how to think about* the value of equity and
   its tax burden.
-  We also cover whether you can [sell private stock](#can-you-sell-private-stock).
+  We also cover whether you can [sell private stock](#
```
Comment (jlevy) [inline README.md]: Broken line.

Comment (Hopeopolis) [reply README.md]: I saw that in the diff and thought Flowmark must have done it and that I should leave it. Did I inadvertently break the line?

Comment (jlevy) [reply README.md]: Can you fix it, flowmark it again and verify? That's not right Markdown so either it's an error in editing or a bug in Flowmark (which you can report to me or report [here](https://github.com/jlevy/atom-flowmark/issues), where we're tracking issues).

File: README.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ legalities is daunting to many individuals, a challenge made even more acute whe
 short window in which to make a key decision, like accepting a
 [job offer](#offers-and-negotiations) or [exercising stock options](#stock-options).
 
-[What to negotiate](#negotiation-tips) and [when to exercise](#stock-option-scenarios) are
-among the decisions individuals have to make that can have **major outcomes** financially.
+[What to negotiate](#negotiation-tips) and [when to exercise](#stock-option-scenarios) are among
+the decisions individuals have to make that can have **major outcomes** financially.
 Because the value of employee equity is determined by the fate of the company, when a
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: This is totally fine — it's just a linewrap!

**Comment by jlevy** [comment]
👍

**Pull request merged**